doc/sphinx/source/backends/index.rst

```diff
@@ -60,3 +61,58 @@ interface; each backend implements the following five functions:
   for later execution.
 * And, finally, the ``call()`` method is used to invoke an nGraph function 
   against a particular set of tensors.
+
+
+.. _miscellaneous_resources: 
+
+Miscellaneous resources
+=======================
+
+Additional resources for device or framework-specific configurations:
+
+OpenCL
```

Comment:
We should note here which backends this is actually relevant for (I think PlaidML, not sure about IntelGPU). This is not a necessary step for CPU backend.

Reply:
Done